test/functional/terminal/scrollback_spec.lua

```diff
@@ -455,8 +454,8 @@ describe("'scrollback' option", function()
 
     retry(nil, nil, function() expect_lines(58) end)
     -- Verify off-screen state
-    eq('line35', eval("getline(line('w0') - 1)"))
-    eq('line26', eval("getline(line('w0') - 10)"))
+    eq((iswin() and 'line36' or 'line35'), eval("getline(line('w0') - 1)"))
+    eq((iswin() and 'line27' or 'line26'), eval("getline(line('w0') - 10)"))
```

Comment:
@janlazo FYI

Reply:
That's fine. Alternative is to expecting `line39` for both and output up to `line39` on Windows only.

Reply:
I think it's a cmd.exe issue. For this case it's not a problem to change the test.